python/dlisio/reprc.py

```diff
@@ -59,13 +59,13 @@
     16     : 'u2',                   #Normal unsigned integer
     17     : 'u4',                   #Long unsigned integer
     18     : 'u4',                   #Variable-length unsigned integer
-    19     : 'i1,U',                 #Variable-length identifier
-    20     : 'i4,U',                 #Variable-length ASCII character string
+    19     : 'u1,U',                 #Variable-length identifier
+    20     : 'u4,U',                 #Variable-length ASCII character string
     21     : 'M',                    #Date and time
-    22     : 'i4',                   #Origin reference
-    23     : 'i4,i1,i1,U',           #Object name
-    24     : 'i1,U,i4,i1,i1,U',      #Object reference
-    25     : 'i1,U,i4,i1,i1,U,i1,U', #Attribute reference
-    26     : 'i1',                   #Boolean status
-    27     : 'i1,U',                 #Units expression
+    22     : 'u4',                   #Origin reference
+    23     : 'u4,u1,u1,U',           #Object name
+    24     : 'u1,U,u4,u1,u1,U',      #Object reference
+    25     : 'u1,U,u4,u1,u1,U,u1,U', #Attribute reference
+    26     : '?',                    #Boolean status
```

Comment:
What's this question mark?

Reply:
It's boolean. (docs.scipy seem to be down, so cached link)
http://webcache.googleusercontent.com/search?q=cache:tyoWStj_4OcJ:https://docs.scipy.org/doc/numpy-1.13.0/reference/arrays.dtypes.html&hl=en&gl=no&strip=1&vwsrc=0
Tests for currently supported representation codes are coming in other PR.

Reply:
Sure, but the logical/boolean in DLIS has a given width, will this always line up? Probably not, and it's quite the risky assumption to make.

Reply:
Numpy uses 1 byte for booleans, as well as DLIS. I doubt this is going to change, so I don't see any risk here. Numpy also doesn't accept size specifier near '?'.
I am not fine with user getting integer value in place of boolean, and this seems the best place to make the conversion.

Reply:
You're right, then it makes perfect sense.